acceptance/acceptance_test.go

```diff
@@ -197,6 +199,24 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 		tmpDir = t.TempDir()
 	}
 
+	cmdEnv := os.Environ()
```

Comment:
You can avoid this extra variable if you move server initialization below.

Reply:
Do you mean directly using `cmd.Env`?

Reply:
yes, keep using cmd.Env as it did before.

Reply:
done